lib/bom.js

```diff
@@ -261,6 +261,61 @@ declare class Worker extends EventTarget {
     terminate(): void;
 }
 
+declare class SharedWorker extends EventTarget {
+    constructor(stringUrl: string): void;
+    port: MessagePort;
+    onerror: (ev: Event) => any;
+}
+
+declare function importScripts(...urls: Array<string>): void;
+
+declare class WorkerGlobalScope extends EventTarget {
+    self: WorkerGlobalScope;
+    location: WorkerLocation;
+    navigator: WorkerNavigator;
+    importScripts(...urls: Array<string>): void;
+    onerror: (ev: Event) => any;
+    onlanguagechange: (ev: Event) => any;
+    onoffline: (ev: Event) => any;
+    ononline: (ev: Event) => any;
+    onrejectionhandled: (ev: PromiseRejectionEvent) => any;
+    onunhandledrejection: (ev: PromiseRejectionEvent) => any;
+}
+
+declare class DedicatedWorkerGlobalScope extends WorkerGlobalScope {
+    close(): void;
```

Comment:
Any reason why `close()` is declared in the subs rather than `WorkerGlobalScope` itself? (FYI I'm using [this](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope) as a reference)